appcli/configure_cli.py

```diff
@@ -83,13 +89,13 @@ def configure(ctx):
             customisation = self.cli_configuration.configure_cli_customisation
 
             logger.debug('Running pre-configure hook')
-            customisation.hooks.pre_configure(cli_context)
+            customisation.hooks.pre_configure_init(cli_context)
             self.__seed_configuration_dir(cli_context)
             # app_configuration_file = cli_context.app_configuration_file
             # app_config_manager = ConfigurationManager(app_configuration_file)
             # self.__configure_all_settings(app_config_manager)
```

Comment:
Let's get rid of these three commented out commands. I don't think we'll use an interactive configurer anymore